src/utils.js

```diff
@@ -219,3 +218,10 @@ export function update(
 
   return set(state, path, next)
 }
+
+/**
+ * A couple of methods use identity functions. This avoids duplication.
+ */
+export function identity<T>(n: T): T {
```

Comment:
Or maybe I could just get away with `identity (n: *) { return n }`. Flow might infer that fine.